README.md

```diff
@@ -20,4 +20,31 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`dixahq/autoassign-reviewers-action`

We need to (roughly) describe what rights are expected from the user token:

`${{ secrets.USER_TOKEN_THAT_CAN_ASSIGN_REVIEWERS_ACTION }}`

Reply:
Will add a comment about the token permission.